atoma-service/src/handlers/image_generations.rs

```diff
@@ -106,8 +114,16 @@ pub async fn image_generations_handler(
     )
     .await
     {
-        Ok(response) => Ok(response),
+        Ok(response) => {
+            TOTAL_COMPLETED_REQUESTS
```

Comment:
I suggest using `model` as the sole label in here

Reply:
a383ece